core-web/apps/dotcms-ui/src/app/view/components/dot-edit-layout-designer/dot-edit-layout-designer.component.ts

```diff
@@ -256,7 +262,7 @@ export class DotEditLayoutDesignerComponent implements OnInit, OnDestroy, OnChan
                     this.currentTheme = theme;
                     this.cd.detectChanges();
                 });
-            this.initialFormValue = _.cloneDeep(this.form.value);
+            this.initialFormValue = _.cloneDeep(this.form.value) as DotTemplateDesigner;
```

Comment:
We have an Issue to [remove lodash](https://github.com/dotCMS/core/issues/24842), maybe we can start here by changing `cloneDeep` for `structuredClone` and remove lodash import from this file if it's possible

Reply:
Yeah, let's start here.